quad_ufabc/Point position
quad_ufabc/Quaternion1 orientation
quad_ufabc/Euler euler_orientation
